src/main/java/org/reaktivity/nukleus/tcp/internal/TcpConfiguration.java

```diff
@@ -32,6 +33,8 @@
 
     private static final int MAX_CONNECTIONS_DEFAULT = Integer.MAX_VALUE;
 
+    private static final boolean TCP_KEEP_ALIVE_DEFAULT = false;
```

Comment:
Rename constant to `KEEPALIVE_DEFAULT` for consistency with naming pattern for other constants.